redfish/resource_redfish_storage_volume.go

```diff
@@ -173,7 +211,21 @@ func createRedfishStorageVolume(service *gofish.Service, d *schema.ResourceData)
 	storageID := d.Get("storage_controller_id").(string)
 	volumeType := d.Get("volume_type").(string)
 	volumeName := d.Get("volume_name").(string)
+	optimumIOSizeBytes := d.Get("optimum_io_size_bytes").(int)
+	capacityBytes := d.Get("capacity_bytes").(int)
 	driveNamesRaw := d.Get("drives").([]interface{})
+	readCachePolicy, ok := d.GetOk("read_cache_policy")
+	if !ok {
+		readCachePolicy = string(redfish.OffReadCachePolicyType)
+	}
+	writeCachePolicy, ok := d.GetOk("write_cache_policy")
+	if !ok {
+		writeCachePolicy = string(redfish.UnprotectedWriteBackWriteCachePolicyType)
```

Comment:
We can use these as default values rather than adding checks

Reply:
added default values to schema